npc/other/goldpc.txt

```diff
@@ -6,50 +6,57 @@
 //= 1.0 Initial release [Lemongrass]
 //= 1.1 Replay version [eppc0330]
 //= 1.2 Translation of the replay version [Lemongrass]
+//= 1.3 Cleanup of NPC logic [Lemongrass]
 //============================================================
 
 prontera,0,0,0	script	Goldpoint Manager::GOLDPCCAFE	4_F_02,{
-	.@point = Goldpc_Points;
-	.@npcName$ = "[Goldpoint Manager]";
-
 	// ID:AMOUNT:PRICE
 	setarray .items$[1],
-	"25464:1:2", // World_Tour_Ticket 1
-	"23919:1:10",  // K_Secret_Key 1
-	"23919:11:100",// K_Secret_Key 11
-	"23919:33:300";// K_Secret_Key 33
+		"25464:1:2", // World_Tour_Ticket 1
+		"23919:1:10",  // K_Secret_Key 1
+		"23919:11:100",// K_Secret_Key 11
+		"23919:33:300";// K_Secret_Key 33
+
+	mes "[Goldpoint Manager]";
+	mes "You currently have ^0000ff"+Goldpc_Points+"^000000 points.";
+	mes "What reward do you want?";
+	next;
 
 	.@menu$ = "View current points";
 
 	for(.@i = 1; .@i < getarraysize(.items$); .@i++) {
 		explode(.@array$, .items$[.@i], ":");
 		.@cost = atoi(.@array$[2]);
-		.@menu$ += ":"+(.@cost)+" points gift"+((.@point<.@cost)?" ^ff0000(not enough points)^000000":"");
+		.@menu$ += ":" + .@cost + " points gift";
```

Comment:
wasn't it supposed to show the item name in the menu? It's just showing the item's price.

Reply:
Check the commit history. The original korean text is included. No item names there. You can modify your own scripts later on however you like. So until proven otherwise it will stay like this.